docs/content/docs/tutorials/rds-example.md

```diff
@@ -256,6 +256,50 @@ spec:
 EOF
 ```
 
+## Restore Snapshot to DBInstance or DBCluster
```

Comment:
I would suggest "Restore a Database Snapshot" as the title. We can cover the semantics of `DBInstace` v `DBCluster` in the language below.